Added in some Words about Battery Drain potential.

CCOSTAN opened the pull request:
Mainly pulled from https://github.com/home-assistant/home-assistant.io/issues/16521 and @mabahj.

## Proposed change
<!-- 
Added in a section about potential battery drain that I found in another issue thread. 
-->

## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 
- https://github.com/home-assistant/home-assistant.io/issues/16521

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

Commit:
Added in some Words about Battery Drain potential.

Mainly pulled from https://github.com/home-assistant/home-assistant.io/issues/16521 and @mabahj.

Commit:
Corrected linting error

Tesla instead of tesla.

frenck commented on source/_integrations/tesla.markdown:
```diff
@@ -51,3 +51,11 @@ This integration provides the following platforms:
 
 * Wake cars on start - Whether to wake sleeping cars on Home Assistant startup. This allows a user to choose whether cars should continue to sleep (and not update information) or to wake up the cars potentially interrupting long term hibernation and increasing vampire drain.
+
+## Potential Battery impacts
+Here are some things to consider and understand when implimenting the Tesla component and it's potential effect on your car's battery. 
+
```
```suggestion
## Potential Battery impacts

Here are some things to consider and understand when implementing the Tesla component and its potential effect on your car's battery.

```

frenck commented on source/_integrations/tesla.markdown:
```diff
@@ -51,3 +51,11 @@ This integration provides the following platforms:
+## Potential Battery impacts
+Here are some things to consider and understand when implimenting the Tesla component and it's potential effect on your car's battery. 
+
+* The `scan_interval` determines when to check if the car is awake and new information is available, but the Tesla integration will not wake up a sleeping car during this polling.  By default, the polling will occur every 660 seconds. Polling a car too frequently can keep the car awake and drain battery.  Different firmware versions and measurements of Tesla cars can take from 11 to 15 minutes fo ...
```
```suggestion
- The `scan_interval` determines when to check if the car is awake and new information is available, but the Tesla integration will not wake up a sleeping car during this polling.  By default, the polling will occur every 660 seconds. Polling a car too frequently can keep the car awake and drain the battery. Different firmware versions and measurements of Tesla cars can take from 11 to 15 minutes for sleep mode to occur. There is no official information on sleep mode timings so your mileage may vary and you should experiment with different polling times for an optimal experience.
```

Additionally, this is not a platform integration, `scan_interval` is not a settable piece of code?

frenck commented on source/_integrations/tesla.markdown:
```diff
@@ -51,3 +51,11 @@ This integration provides the following platforms:
+Here are some things to consider and understand when implimenting the Tesla component and it's potential effect on your car's battery. 
+
+* The `scan_interval` determines when to check if the car is awake and new information is available, but the Tesla integration will not wake up a sleeping car during this polling.  By default, the polling will occur every 660 seconds. Polling a car too frequently can keep the car awake and drain battery.  Different firmware versions and measurements of Tesla cars can take from 11 to 15 minutes fo ...
+* The car will, however, be woken up when a command is actively sent to the car, such as door unlock or turning on the HVAC. It will then also fetch updated information while the car is awake based on the `scan_interval`.
```
```suggestion
- The car will, however, be woken up when a command is actively sent to the car, such as door unlock or turning on the HVAC. It will then also fetch updated information while the car is awake based on the `scan_interval`.
```

Same comment as above about the `scan_interval`.

frenck commented on source/_integrations/tesla.markdown:
```diff
@@ -51,3 +51,11 @@ This integration provides the following platforms:
+
+* The `scan_interval` determines when to check if the car is awake and new information is available, but the Tesla integration will not wake up a sleeping car during this polling.  By default, the polling will occur every 660 seconds. Polling a car too frequently can keep the car awake and drain battery.  Different firmware versions and measurements of Tesla cars can take from 11 to 15 minutes fo ...
+* The car will, however, be woken up when a command is actively sent to the car, such as door unlock or turning on the HVAC. It will then also fetch updated information while the car is awake based on the `scan_interval`.
+* The car can intentionally be woken up to fetch recent information by sending a harmless command, for example a lock command. This can be used in an automation to, for example, ensure that updated information is available every morning. (Note that the command must be valid for that specific car model. So locking the frunk of a Model 3 will not wake up that car).
```
```suggestion
- The car can intentionally be woken up to fetch recent information by sending a harmless command, for example, a lock command. This can be used in an automation to, for example, ensure that updated information is available every morning. (Note that the command must be valid for that specific car model. So locking the frunk of a Model 3 will not wake up that car).
```

frenck commented on source/_integrations/tesla.markdown:
```diff
@@ -51,3 +51,11 @@ This integration provides the following platforms:
+* The `scan_interval` determines when to check if the car is awake and new information is available, but the Tesla integration will not wake up a sleeping car during this polling.  By default, the polling will occur every 660 seconds. Polling a car too frequently can keep the car awake and drain battery.  Different firmware versions and measurements of Tesla cars can take from 11 to 15 minutes fo ...
+* The car will, however, be woken up when a command is actively sent to the car, such as door unlock or turning on the HVAC. It will then also fetch updated information while the car is awake based on the `scan_interval`.
+* The car can intentionally be woken up to fetch recent information by sending a harmless command, for example a lock command. This can be used in an automation to, for example, ensure that updated information is available every morning. (Note that the command must be valid for that specific car model. So locking the frunk of a Model 3 will not wake up that car).
+* You can also use the `Update switch` to programatically disable polling of the vehicle completely.
```
```suggestion
- You can also use the `Update switch` to programmatically disable polling of the vehicle completely.
```

What does this mean? And how does the reader knows what this means?

Commit:
Update source/_integrations/tesla.markdown

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

Commit:
Update source/_integrations/tesla.markdown

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

Commit:
Update source/_integrations/tesla.markdown

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

Commit:
Update source/_integrations/tesla.markdown

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

Commit:
Update source/_integrations/tesla.markdown

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

Commit:
Make the scan_interval phrase clearer.

Added in the other option and referred to it as the scan_interval to make the description clearer hopefully.
